### ISSUE TITLE
C3 rt 116 Possible performance regression with regard to fill rate

### ISSUE THREAD

**Issue by NetOneGames** [Scirra/Construct-3-bugs#2023]
## Problem description

Check the Fill rate Tech Demo 

Try in C2 rt and C3 115 and observe similar performance.

Try C3  rt 116 and abserve much worse performance.

## Attach a .c3p

Use Fill Rate tech demo on start page.

* Chrome: tested only

## System details

<details><summary>View details</summary>

Platform information
Browser: Chrome
Browser version: 68.0.3440.75
Browser engine: Blink
Browser architecture: 64-bit
Context: browser
Operating system: Windows
Operating system version: 7
Operating system architecture: 64-bit
Device type: desktop
Device pixel ratio: 1.25
Logical CPU cores: 8
Approx. device memory: 8 GB
User agent: Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.75 Safari/537.36
C3 release: r116 (beta)
Language setting: en-US

Local storage
Storage quota (approx): 47 gb
Storage usage (approx): 574 mb (1.2%)
Persistant storage: Yes

Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.

WebGL 2+ is not supported. Rendering quality and features may be affected.
WebGL indicates a major performance caveat. It is probably using software rendering.
WebGL information
Version string: WebGL 1.0 (OpenGL ES 2.0 Chromium)
Numeric version: 1
Supports NPOT textures: partial
Supports GPU profiling: no
Vendor: Google Inc.
Renderer: Google SwiftShader
Major performance caveat: yes
Maximum texture size: 8192
Point size range: 0.125 to 2048
Extensions:

ANGLE_instanced_arrays
EXT_blend_minmax
EXT_color_buffer_half_float
EXT_texture_filter_anisotropic
WEBKIT_EXT_texture_filter_anisotropic
OES_element_index_uint
OES_standard_derivatives
OES_texture_float
OES_texture_float_linear
OES_texture_half_float
OES_texture_half_float_linear
OES_vertex_array_object
WEBGL_color_buffer_float
WEBGL_compressed_texture_etc1
WEBGL_compressed_texture_s3tc
WEBKIT_WEBGL_compressed_texture_s3tc
WEBGL_debug_renderer_info
WEBGL_debug_shaders
WEBGL_depth_texture
WEBKIT_WEBGL_depth_texture
WEBGL_draw_buffers
WEBGL_lose_context
WEBKIT_WEBGL_lose_context
Audio information
System sample rate: 48000 Hz
Output channels: 2
Output interpretation: speakers
Supported decode formats:

Ogg Opus (audio/ogg; codecs=opus)
WebM Opus (audio/webm; codecs=opus)
Ogg Vorbis (audio/ogg; codecs=vorbis)
WebM Vorbis (audio/webm; codecs=vorbis)
MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats:

WebM Opus (audio/webm; codecs=opus)
Video information
Supported decode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
Ogg Theora (video/ogg; codecs=theora)
H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)

</details>

**Comment by AshleyScirra** [comment]
No, but this is nothing to do with Construct so we can't help with it, it's decided entirely by the browser or GPU hardware.

**Comment by NetOneGames** [comment]
yes you are right :   with hardware enabled same for both.    please give me a slap......  I will shut up now....
